feat: added evm_setBlockGasLimit (#332)
matter-labs/era-test-node: SUPPORTED_APIS.md

| `EVM` | `evm_setAccountNonce` | `NOT IMPLEMENTED` | Sets the given account's nonce to the specified value |
| `EVM` | `evm_setAccountStorageAt` | `NOT IMPLEMENTED` | Sets the given account's storage slot to the specified data |
| `EVM` | `evm_setAutomine` | `NOT IMPLEMENTED` | Enables or disables the automatic mining of new blocks with each new transaction submitted to the network |

> | `EVM` | [`evm_setBlockGasLimit`](#evm_setblockgaslimit) | `SUPPORTED` | Sets the Block Gas Limit of the network |

Reviewer: ```suggestion
| `EVM` | [`evm_setBlockGasLimit`](#evm_setblockgaslimit) | `SUPPORTED` | Sets the block gas limit for the following blocks |
```

  --data '{"jsonrpc": "2.0","id": "1","method": "evm_revert","params": ["0x1"]}'
```

### `evm_setBlockGasLimit`

[source](src/node/evm.rs)

> Sets the Block Gas Limit of the network

Reviewer: ```suggestion
Sets the block gas limit for the following blocks
```
